src/plugins/kibana_react/public/theme/wrap_with_theme.tsx

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { Observable } from 'rxjs';
+import type { CoreTheme } from '../../../../core/public';
+import { KibanaThemeProvider } from './kibana_theme_provider';
+
+export const wrapWithTheme = (
+  node: React.ReactNode,
+  theme$: Observable<CoreTheme>
+): React.ReactElement => {
+  return <KibanaThemeProvider theme$={theme$}>{node}</KibanaThemeProvider>;
+};
```

Comment:
I'm currently exporting both `<KibanaThemeProvider>` and `wrapWithTheme()` from `kibana_react`, even if both allow to do the exact same thing, the function being basically just a decorator for the provided element. I wonder if I should delete `wrapWithTheme` and only expose the component, wdyt?

Reply:
The decorator pattern doesn't seem to be widely used in Kibana, I'd delete it. One can implement it locally if necessary.